portfolio/src/main/java/com/google/sps/utils/ObjectDataStore.java

```diff
@@ -0,0 +1,18 @@
+package com.google.sps.utils;
+
+import java.util.List;
+
+/** Interface that provides the service to interact with Datastore to store or load object. */
+public interface ObjectDataStore<T> {
+    /**
+     * Puts the entity into Datastore.
```

Comment:
Nit: object instead of entity, same for below method

Reply:
Fixed